src/services/formatting/formatting.ts

```diff
@@ -501,9 +501,6 @@ namespace ts.formatting {
                 else if (SmartIndenter.argumentStartsOnSameLineAsPreviousArgument(parent, node, startLine, sourceFile)) {
                     return { indentation: parentDynamicIndentation.getIndentation(), delta };
                 }
-                else if (SmartIndenter.isJSXClosingWithMultiLineText(parent, node, startLine, sourceFile)) {

```

Comment:
So this is where we were trying to compensate for the old bad scanner behavior; we'd essentially _dedent_ (or at least choose not to increase the indent) in this case to try to compensate for the extra whitespace the scanner used to provide.